WordPress/src/main/java/org/wordpress/android/util/NetworkUtilsWrapper.kt

```diff
@@ -5,7 +5,7 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class NetworkUtilsWrapper @Inject constructor() {
+open class NetworkUtilsWrapper @Inject constructor() {
```

Comment:
ℹ️ This does not need to be `open` at the moment.

Reply:
Removed in 062cd3be87c736fff6d141658c2bb338604dc50c